src/backend/Dot.ml

```diff
@@ -66,38 +78,41 @@ module Make(S : Msat.S)(A : Arg with type atom := S.atom
     let v = Clause.atoms c in
     if Array.length v = 0 then
       Format.fprintf fmt "⊥"
-    else
+    else (
       let n = Array.length v in
       for i = 0 to n - 1 do
         Format.fprintf fmt "%a" A.print_atom v.(i);
         if i < n - 1 then
           Format.fprintf fmt ", "
-      done
+      done )
 
-  let print_edge fmt i j =
-    Format.fprintf fmt "%s -> %s;@\n" j i
+  let print_edge fmt i j = Format.fprintf fmt "%s -> %s;@\n" j i
 
   let print_edges fmt n =
     match P.(n.step) with
-    | P.Hyper_res {P.hr_steps=[];_} -> assert false (* NOTE: should never happen *)
-    | P.Hyper_res {P.hr_init; hr_steps=((_,p0)::_) as l} ->
+    | P.Hyper_res {P.hr_steps = []; _} ->
+      assert false (* NOTE: should never happen *)
+    | P.Hyper_res {P.hr_init; hr_steps = (_, p0) :: _ as l} ->
```

Comment:
Is there an ocamlformat option to have spaces after the opening brace and before the closing one ?

Reply:
personally I prefer this style. I think there might be an option but I find it too sparse :woman_shrugging:

Reply:
I'll try and look at it on a non-laptop screen when I can to see if I can get used to it, ^^